packages/components/toggle/toggle.scss

```
@@ -50,13 +50,48 @@
             radius: 8px;
         }
 
+        transition: background 100ms ease-in-out;
         height: var(--kbq-toggle-size-normal-height);
         width: var(--kbq-toggle-size-normal-width);
+
+        .kbq-toggle__thumb {
+            position: absolute;
+            left: 8px;
```

мб переменные использовать?

```suggestion
            left: var(--kbq-size-s);
```

---

переделал на sass переменную, вроде выглядит понятнее